defaults/main.yml

```diff
@@ -61,8 +63,8 @@ acas_instance_subnet:
 
 termination_protection: true
 
-ssh_via_private_ip: false
+ssh_via_private_ip: true
```

Comment:
Just noting that this is a change to the default too but I think it's a good change. It makes more sense for the script to use the private IP as it covers the case where we haven't setup dns entries for the new instance.